pkg/pillar/scripts/device-steps.sh

```diff
@@ -54,6 +55,18 @@ else
     USE_HW_WATCHDOG=0
 fi
 
+LOOP_COUNT=0
+while [ -z $(pgrep logread) ];
+do
+    sleep 1
+    LOOP_COUNT=`expr $LOOP_COUNT + 1`
+    if [ $LOOP_COUNT -ge $LOGREAD_PID_WAIT ]; then
+        echo "Error: Could not find logread process"
```

Comment:
Nit, but it would be good to have the timestamp here as we have for the other echo log commands.

Reply:
Addressed your comment and committed code.